modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/expression/FilterByCountIndividualSegmentsExpressionVisitorImpl.java

```diff
@@ -112,6 +114,61 @@ else if (Objects.equals("day", value)) {
 			"Unsupported operation with value " + value);
 	}
 
+	@Override
+	public FilterByCount.Day visitFunctionCallExpression(
+		@NotNull
+			IndividualSegmentsExpressionParser.FunctionCallExpressionContext
+				functionCallExpressionContext) {
+
+		ParseTree parseTree1 = functionCallExpressionContext.getChild(0);
+
+		Object functionName = parseTree1.accept(this);
```

Comment:
you can use parseTree1.accept(this) inline in Objects.equals("between", functionName)

Reply:
Yes, well seen, thanks !

Reply:
No, because I am using the functionName some lines below when I throw an Exception if we are not allowing the function.